### PR TITLE
[doc] add more description for music skill

### PR DESCRIPTION
**Pull request by xhr0804**
### Purpose
*What is the context of this pull request? Why is it being done?*
related to https://github.com/microsoft/botframework-skills/issues/75
### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
Music skill is too simple to add transcript, so just add more description
### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: docs/_docs/skills/samples/music.md [hunk 1]
```diff
@@ -14,6 +14,14 @@ toc: true
 The [Music skill]({{site.repo}}/tree/master/skills/csharp/experimental/musicskill) integrates with [Spotify](https://developer.spotify.com/documentation/web-api/libraries/) to look up playlists and artists and open the Spotify app via URI.
 This is dependent on the [SpotifyAPI-NET](https://github.com/JohnnyCrazy/SpotifyAPI-NET) wrapper for the Spotify Web API.
 
+This skill has a very limited LUIS model (available in English and Chinese) and demonstrates a simple scenarios:
```
Comment (darrenj) [inline docs/_docs/skills/samples/music.md:17]: We now have a fully fledged LUIS model - can we update to this?

**Review by darrenj — CHANGES_REQUESTED** [review]

File: docs/_docs/skills/samples/music.md (hunk 1 above)
Comment (xhr0804) [reply docs/_docs/skills/samples/music.md:17]: sure, open an issue to track the task, and will update the model soon. https://github.com/microsoft/botframework-skills/issues/121

Comment (lzc850612) [reply docs/_docs/skills/samples/music.md:17]: are you ready to make the change now?

Comment (xhr0804) [reply docs/_docs/skills/samples/music.md:17]: will discuss more on the new luis model, so close this PR first.

**Comment by xhr0804** [comment]
will update the luis model and submit a new PR.

**Pull request closed without merging**